endpointmanager/scripts/populatenpiorgs.go

```diff
@@ -0,0 +1,423 @@
+	Provider_Organization_Name_Legal_Business_Name                         string
+	Provider_Last_Name_Legal_Name                                          string
+	Provider_First_Name                                                    string
+	Provider_Middle_Name                                                   string
+	Provider_Name_Prefix_Text                                              string
+	Provider_Name_Suffix_Text                                              string
+	Provider_Credential_Text                                               string
+	Provider_Other_Organization_Name                                       string
+	Provider_Other_Organization_Name_Type_Code                             string
+	Provider_Other_Last_Name                                               string
+	Provider_Other_First_Name                                              string
+	Provider_Other_Middle_Name                                             string
+	Provider_Other_Name_Prefix_Text                                        string
+	Provider_Other_Name_Suffix_Text                                        string
+	Provider_Other_Credential_Text                                         string
+	Provider_Other_Last_Name_Type_Code                                     string
+	Provider_First_Line_Business_Mailing_Address                           string
+	Provider_Second_Line_Business_Mailing_Address                          string
+	Provider_Business_Mailing_Address_City_Name                            string
+	Provider_Business_Mailing_Address_State_Name                           string
+	Provider_Business_Mailing_Address_Postal_Code                          string
+	Provider_Business_Mailing_Address_Country_Code_If_outside_US           string
+	Provider_Business_Mailing_Address_Telephone_Number                     string
+	Provider_Business_Mailing_Address_Fax_Number                           string
+	Provider_First_Line_Business_Practice_Location_Address                 string
+	Provider_Second_Line_Business_Practice_Location_Address                string
+	Provider_Business_Practice_Location_Address_City_Name                  string
+	Provider_Business_Practice_Location_Address_State_Name                 string
+	Provider_Business_Practice_Location_Address_Postal_Code                string
+	Provider_Business_Practice_Location_Address_Country_Code_If_outside_US string
+	Provider_Business_Practice_Location_Address_Telephone_Number           string
+	Provider_Business_Practice_Location_Address_Fax_Number                 string
+	Provider_Enumeration_Date                                              string
+	Last_Update_Date                                                       string
+	NPI_Deactivation_Reason_Code                                           string
+	NPI_Deactivation_Date                                                  string
+	NPI_Reactivation_Date                                                  string
```

Comment:
thoughts on including NPI activation status? Perhaps we don't parse deactivated entities? that would explain why there are a jillion of them - the db keeps everything and just tags them as deactivated.

Reply:
I don't think the NPI data source stores deactivated entries. IE: In my search I didn't find any entries that had a Deactivation Date but no Reactivation Date